### ISSUE TITLE
Does libsumo support Controlling parallel simulations from the same TraCI script?

### ISSUE THREAD

**Issue by Trashcleaner** [eclipse/sumo#7694]
I am trying to run multiple simulations via TraCI in parallel as [documented on SUMO wiki](https://sumo.dlr.de/docs/TraCI/Interfacing_TraCI_from_Python.html#controlling_parallel_simulations_from_the_same_traci_script). 

It seems working well when I `import traci`.  However, is it somehow possible to `import libsumo as traci` to get even better speedup?

The following code:
```python
# import traci
import libsumo as traci

# ...

def startParallelSimulations(sumoCommands, labels):
    # ...
    for sumoCmd, label in zip(sumoCommands, labels):
        traci.start(sumoCmd, label=label)
    # ...
```
throws a `TypeError`:
```
    ....
    traci.start(sumoCmd, label=label) 
TypeError: start() got an unexpected keyword argument 'label'   
```

So it seems `libsumo.start` doesn't accept `label` parameter.
Is there another way?

Thank you very much.

**Comment by behrisch** [comment]
No, libsumo links sumo directly and you cannot run multiple simulations in the same sumo run. You might be able to run one simulation with libsumo and some others with traci from the same script or you could try the [multiprocessing module](https://docs.python.org/3/library/multiprocessing.html) from python which should fork different processes. If you try this please report back whether it works.

**Comment by Trashcleaner** [comment]
Thank you for the advice. It seems working ok. I created a minimal example with the Python multiprocessing module. The code looks something like this:

```python
# ...
from multiprocessing import Process
import libsumo as traci
# ...

def runScenario(sumoCmd, label):
    traci.start(sumoCmd)
    step = 1
    while step <= 1000:
        traci.simulationStep()
        if label == 'label1':
            do_something()
        elif label == 'label2':
            do_something_else()
        step += 1
    traci.close()

# ...

def main():
    sumoBinary = sumolib.checkBinary('sumo')

    sumoCmd1 = [sumoBinary, "-c", "scenario1.sumocfg", "-b", "0"]
    sumoCmd2 = [sumoBinary, "-c", "scenario2.sumocfg", "-b", "0"]

    p1 = Process(target=runScenario, args=(sumoCmd1, 'label1', ))
    p2 = Process(target=runScenario, args=(sumoCmd2, 'label2', ))
    p1.start()
    p2.start()
    p1.join()
    p2.join()
```
This runs two simulations in parallel and correctly produces outputs into respective directories specified in `.sumocfg` files. I will be able to automate my parametrized simulations more quickly using this approach.

If I encounter some problems using more advanced simulations, I will report them here or enter new issue.
Thanks.
